test/integration/targets/ansible-config/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+- name: test ansible-config for valid output and no dupes
+  block:
+    - name: Create temporary file
+      tempfile:
+        path: '{{output_dir}}'
+        state: file
+        suffix: temp.ini
+      register: ini_tempfile
+
+    - name: run config full dump
+      shell: ansible-config init -t all > {{ini_tempfile.path}}
+
+    - name: run ini tester
+      shell: "{{ansible_playbook_python}} '{{role_path}}/files/ini_dupes.py' '{{ini_tempfile.path}}'"
+      ignore_errors: true
```

Comment:
Unless we are asserting the output we should be able to remove the assert check and this line